_editorial_policies/03_reviewer_information.md

```diff
@@ -0,0 +1,111 @@
+
+
+Because we are attempting to make LiveCoMS as open as possible, we provide examples below of typical form letters used for correspondence with reviewers.
+
+## Reviewer letters
+
+### Sample review request letter
+
+Dear XYZ,
+
+We invite you to review the article indicated below, which has been submitted to the `YYY` category of the [Living Journal of Computational Molecular Science](http://livecomsjournal.org) (LiveCoMS).
+LiveCoMS is a brand new and rather unique journal, which focuses on promoting best practices in molecular computation via updateable educational articles of various kinds - tutorials, reviews, best-practices “manuals” and “lessons learned” articles.
+The journal is low-cost, non-profit and run by members of our community in a transparent way; in part, it aims to provide academic credit for important work that often goes unrewarded.
+One unique feature of the review process is the requirement for a student review of every article.
+
+To ACCEPT: `link`
+To DECLINE: `link`
+
+Reviews are generally expected within 10 days, although extra time may be allowable upon request.
+
+Please be sure you review and understand our [conflict of interest policies](FILL IN LINK) before making a decision on whether or not to accept this review request.
+
+If you have any questions, please reply to this email.
+
+Thank you very much.
+`{{SENDER-FULL-NAME}}`, Associate Editor
+
+`{{MANUSCRIPT-TITLE}}`
+Abstract:
+(abstract will be automatically included)
+
+
+### Sample review action/thanks for agreeing letter
+
+(Each category has its own form letter, differing in the link to category-specific review criteria. The below sample is for Best Practices Guides)
+
+Dear XYZ,
+
+Thank you very much for agreeing to review the LiveCoMS submission, `{{MANUSCRIPT-TITLE}}`.
+Your support of our new community-run journal is very encouraging.
+
+I would appreciate receiving your review within 10 days, by `TTTTTT`.
+
+The article is to be considered under the `YYY` category.
+To aid you in your review, both the general [review criteria](https://livecomsjournal.github.io/authors/policies/#review-criteria) and the criteria specific to [Best Practices Guides](https://livecomsjournal.github.io/authors/best_practices/) are appended below.
+
+Please read our conflict-of-interest criteria and inform me of any potential conflicts.
+
+Note also that non-anonymous feedback can be provided directly to authors via the GitHub page linked in the manuscript, if you so desire.
+
+If you have any questions, please reply to this email.
+
+Thank you very much.
+`{{SENDER-FULL-NAME}}`, Associate Editor
+
+REVIEW CRITERIA
+`Fill in current general review criteria`
+`Fill in current category-specific review criteria`
+
+### Reviewer ranking questions
+
+(All fields will be communicated to the authors except comments to the editor)
+- Overall rating (one to five stars)
+- For this manuscript I recommend (Accept/Revise and resubmit/Reject)
```

Comment:
major vs. minor revision

Reply:
Again, I THINK this is required Scholastica terminology -- I don't think we can add options here. At least, that's what their form seems to indicate.

I'll leave as is for now but if @mrshirts knows otherwise we can adjust, or if we find out we have more flexibility once we start seeing submissions we can adjust.